apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -257,7 +259,7 @@ export default class AnimationPickerBody extends React.Component {
         <h1 style={dialogStyles.title}>
           {!animationJsonMode && msg.animationPicker_title({assetType})}
         </h1>
-        {showDrawAndUploadButtons && (
```

Comment:
should the comment on `showDrawAndUploadButtons` be modified to indicate it doesn't necessarily mean we will show the upload button?

Reply:
Updated!